src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/SqlConnectionX.cs

```diff
@@ -32,8 +32,15 @@ internal sealed class SqlConnectionX : DbConnection, ICloneable
         private SqlDataSource? _dataSource;
         private SqlConnector? _internalConnection;
 
+        bool _disposed;
+
+        internal SqlConnector? InternalConnection { get => _internalConnection; }
+
         //TODO: Investigate if we can just use dataSource.ConnectionString. Do this when this class can resolve its own data source.
         private string _connectionString = string.Empty;
+        internal SqlConnectionStringBuilder Settings { get; private set; } = DefaultSettings;
+        static readonly SqlConnectionStringBuilder DefaultSettings = new();
```

Comment:
Readonly members should go at the top of the class, properties should go after constructors